pandas/tests/series/indexing/test_setitem.py

```diff
@@ -126,6 +127,13 @@ def test_setitem_boolean_different_order(self, string_series):
 
         tm.assert_series_equal(copy, expected)
 
+    def test_setitem_boolean_ea_type(self):
+        # GH: 26468
```

Comment:
can you also replicate the `.loc` case in the OP

Reply:
Yes, done.

Reply:
i think what jreback was asking for here is to do `ser.loc[mask] = ` in addition to `ser[mask] = `, not putting `ser.loc[mask]` on the RHS.

Reply:
Misunderstood this, adjusted the test

Reply:
ok this *should* also work for Float EA types, I think we need a fixture that is any_numeric_dtype == any_nullable_int_dtype + float_ea_dtype

Reply:
Added the fixture. Tests pass locally.